bin/faucet/Cargo.toml

```diff
@@ -47,6 +47,7 @@ miden-node-utils = { workspace = true, features = ["vergen"] }
 static-files     = "0.2"
 
 [dev-dependencies]
-fantoccini   = { version = "0.21" }
-serde_json   = { version = "1.0" }
-tokio-stream = { workspace = true, features = ["net"] }
+fantoccini    = { version = "0.21" }
+serde_json    = { version = "1.0" }
+tokio-stream  = { workspace = true, features = ["net"] }
+miden-testing = { git = "https://github.com/0xMiden/miden-base.git", branch = "next" }
```

Comment:
This should now ideally reference [the `0.9` version of this crate](https://github.com/0xMiden/miden-base/releases)